### PR TITLE
Enhancement: Normalize composer.json

### PR DESCRIPTION
**Pull request by localheinz**
| Q             | A
| ------------- | ---
| Branch?       | master for features / 2.7 up to 4.0 for bug fixes 
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | n/a
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

This PR

* [x] normalizes `composer.json`

Ran

```
$ composer normalize && find src -name composer.json -exec composer normalize "./{}" \;
```

For reference, see https://github.com/localheinz/composer-normalize.

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/composer.json [hunk 1]
```diff
@@ -20,24 +20,13 @@
         "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
         "php": ">=5.3.3"
```
Comment (localheinz) [inline src/Symfony/Bridge/PhpUnit/composer.json]: Manually reverted this block (because of duplicated `php` keys).

**Comment by nicolas-grekas** [comment]
I'm sorry but I'm :-1: @localheinz: composer.json files are already hard to merge from 2.7 to master when any of them changes, this will make it a nightmare.

**Comment by Simperfit** [comment]
I'm 👎 too since it does not bring that much values for now.

**Comment by fabpot** [comment]
ok, let's close then.

**Comment by localheinz** [comment]
@fabp @nicolas-grekas @Simperfit 

No problem, maybe you will find this useful at another time. I believe consistent structuring of `composer.json` will make the merge process easier, eventually.

**Pull request closed without merging**